mautic/mautic
# [UX] Email theme: Attract

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | 🔴 
| New feature/enhancement? (use the a.x branch)      | 🟢
| Deprecations?                          | 🔴
| BC breaks? (use the c.x branch)        | 🔴
| Automated tests included?              | 🔴 
| Related user documentation PR URL      | mautic/user-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation-new#... 
| Issue(s) addressed                     | Fixes #... 



## Description

This PR adds a new email theme.

![screencapture-127-0-0-1-32772-email-preview-35-2024-10-15-10_04_50](https://github.com/user-attachments/assets/195f673f-28a8-4840-9ea5-3cfc231329f6)






---
### 📋 Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open Emails
3. Click to create New
4. Find and select the Attract theme
5. Open the builder and see
6. Click to save then Preview
7. Look if the preview looks like in the builder
